src/api/app/jobs/populate_to_sphinx_job.rb

```diff
@@ -0,0 +1,8 @@
+class PopulateToSphinxJob < ApplicationJob
+  def perform(id:, model_name:)
+    object = model_name.to_s.camelize.safe_constantize.find(id)
```

Comment:
Maybe...

```suggestion
    object = model_name == 'Project' ? Project.find(id) : Package.find(id)
```

Reply:
I guess we should also guard about the record being gone...

Reply:
Done